src/standalone/Snow/Snow.jl

```diff
@@ -253,20 +282,19 @@ function ClimaLand.make_update_aux(model::SnowModel{FT}) where {FT}
 
         @. p.snow.T_sfc = snow_surface_temperature(p.snow.T)
 
-        p.snow.turbulent_fluxes .= turbulent_fluxes(model.atmos, model, Y, p, t)
-
-        p.snow.R_n .= net_radiation(model.radiation, model, Y, p, t)
-
         @. p.snow.water_runoff =
             compute_water_runoff(Y.snow.S, p.snow.q_l, p.snow.T, parameters)
 
         @. p.snow.energy_runoff =
             p.snow.water_runoff * volumetric_internal_energy_liq(FT, parameters)
+        @. p.snow.snow_cover_fraction = scf(Y.snow.S)
     end
 end
 
-function ClimaLand.make_compute_exp_tendency(model::SnowModel{FT}) where {FT}
-    function compute_exp_tendency!(dY, Y, p, t)
+function ClimaLand.make_update_boundary_fluxes(model::SnowModel{FT}) where {FT}
+    function update_boundary_fluxes!(p, Y, t)
+        p.snow.turbulent_fluxes .= turbulent_fluxes(model.atmos, model, Y, p, t)
```

Comment:
This function could care about other components. For example, the net radiation will change if the snow is under a canopy.

Reply:
(we would account for that using dispatch in `net_radiation` or similar)